packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -170,6 +176,16 @@ export class CdkToolkit {
 
         if (Object.keys(result.outputs).length > 0) {
           print('\nOutputs:');
+
+          if (outputsPath) {
+            const outputs = fs.readJSONSync(outputsPath);
```

Comment:
Why do we need to read the file first? I would expect this to simply override with the new outputs...

Reply:
it does - but when deploying a wild card stack or multiple stacks, since the outputs are written at the end of every `deploy`, it would wipe out outputs from previous stack if we don't read and append

Reply:
alternatively, we could always write at the very end... and if a stack fails deployment, we could write what we have so far before we throw. I do like that more than the idea of reading a lot. I'll make that change and we'll see whether we like that more!

Reply:
got rid of this